cpp/doxygen/main_page.md

```diff
@@ -0,0 +1,13 @@
+# libraft
+
+RAFT is an analytics engine, containing building-blocks for rapid composition of RAPIDS Analytics. These building-blocks include shared representations, mathematical computational primitives, and utilities that accelerate building analytics and data science algorithms in the RAPIDS ecosystem. Both the C++ and Python components can be included in consuming libraries, providing building-blocks for both dense and sparse matrix formats in the following general categories:
```

Comment:
Can we maybe use a different term on the third "building-blocks"? Feels like a little repetitive term, we could use "representations"?

Reply:
Maybe not referring to it as an "engine" which sounds more like a server or executable to me? Library, framework, collection, building blocks, those are all great terms.